src/Event/ReadModel/JSONLD/CdbXMLImporter.php

```diff
@@ -597,8 +597,17 @@ private function importCalendar(\CultureFeed_Cdb_Item_Event $event, $jsonLD)
     private function importTypicalAgeRange(\CultureFeed_Cdb_Item_Event $event, $jsonLD)
     {
         $ageFrom = $event->getAgeFrom();
-        if ($ageFrom) {
-            $jsonLD->typicalAgeRange = "{$ageFrom}-";
+
+        if (isset($ageFrom) && is_int($ageFrom)) {
+            if ($ageFrom <= 12) {
+                $jsonLD->typicalAgeRange = "{$ageFrom}-12";
+            } else if ($ageFrom <= 18) {
+                $jsonLD->typicalAgeRange = "{$ageFrom}-18";
+            } else if ($ageFrom <= 99) {
+                $jsonLD->typicalAgeRange = "{$ageFrom}-99";
+            } else if ($ageFrom === 101) {
```

Comment:
Does this business rule only apply specificly to 101 or should it be everything bigger than 99?